lib/view_model/single_page/daily/daily_view_model.dart

```diff
@@ -1,59 +1,54 @@
 import 'package:riverpod_annotation/riverpod_annotation.dart';
 
-import 'package:hakondate/model/dictionary/dictionary_item_model.dart';
 import 'package:hakondate/model/dish/dish_model.dart';
 import 'package:hakondate/model/menu/menu_model.dart';
-import 'package:hakondate/model/nutrients/five_major_nutrient.dart';
-import 'package:hakondate/model/nutrients/nutrients_model.dart';
-import 'package:hakondate/repository/local/sqlite/dictionary_items/dictionary_items_local_repository.dart';
 import 'package:hakondate/repository/local/sqlite/menus/menus_local_repository.dart';
 import 'package:hakondate/state/daily/daily_state.dart';
 import 'package:hakondate/util/analytics_controller/analytics_controller.dart';
 import 'package:hakondate/util/environment.dart';
-import 'package:hakondate/view_model/multi_page/user/user_view_model.dart';
 
 part 'daily_view_model.g.dart';
 
 @Riverpod(keepAlive: true)
 class DailyViewModel extends _$DailyViewModel {
   @override
-  FutureOr<DailyState> build() {
+  Future<DailyState> build() async {
+    DateTime? selectedDay;
+    switch (Environment.flavor) {
+      case Flavor.dev:
+        selectedDay ??= await ref.read(menusLocalRepositoryProvider).getLatestDay();
+      case Flavor.stg || Flavor.prod:
+        selectedDay ??= DateTime.now();
+    }
+    final MenuModel menu = await ref.read(menusLocalRepositoryProvider).getMenuByDay(selectedDay);
+
     return DailyState(
-      selectedDay: DateTime.now(),
-      focusedDay: DateTime.now(),
-      calendarTabFirstDay: DateTime(2019, 8),
+      selectedDay: selectedDay,
+      focusedDay: selectedDay,
+      calendarTabFirstDay: await ref.read(menusLocalRepositoryProvider).getOldestDay(),
       calendarTabLastDay: DateTime(
         DateTime.now().year,
         DateTime.now().month + 2,
       ).add(const Duration(seconds: -1)),
+      menu: menu,
     );
   }
 
-  Future<void> updateSelectedDay({DateTime? selectedDay, DateTime? focusedDay}) async {
+  Future<void> updateSelectedDay(DateTime selectedDay, {DateTime? focusedDay}) async {
```

Comment:
引数の形変える必要ないかな

Reply:
updateSelectedDayっていう関数名なので引数がselected dayだということは自明かなと思ったのと，他のupdateHoge関数の仕様と近づけてみたかったという意図でした．
変える必要はないと思いつつ，逆に変えない方がいい理由ってありましたっけ？

Reply:
selectedDayがnullableである必要がなくなったので，こんな感じにするのはありそう.
```suggestion
  Future<void> updateSelectedDay({required DateTime selectedDay, DateTime? focusedDay}) async {
```